ui/admin/app/components/form/target/details/index.js

```diff
@@ -2,20 +2,24 @@ import Component from '@glimmer/component';
 import { TYPES_TARGET } from 'api/models/target';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
 
 // NOTE: this is all a temporary solution till we have a resource type helper.
 const types = [...TYPES_TARGET].reverse();
 const icons = {
   ssh: 'terminal-screen',
   tcp: 'network',
 };
-
+const workerFilters = ['egress_worker_filter', 'ingress_worker_filter'];
 export default class FormTargetComponent extends Component {
   // =properties
   @tracked egressWorkerFilterEnabled =
     this.args.model.egress_worker_filter?.length;
+  @tracked ingressWorkerFilterEnabled =
+    this.args.model.ingress_worker_filter?.length;
```

Comment:
These are strings and an empty string is falsy, so testing length should be unnecessary.
```suggestion
  @tracked egressWorkerFilterEnabled = this.args.model.egress_worker_filter;
  @tracked ingressWorkerFilterEnabled = this.args.model.ingress_worker_filter;
```